packages/auth-providers/clerk/api/src/decoder.ts

```diff
@@ -34,3 +37,36 @@ export const authDecoder: Decoder = async (token: string, type: string) => {
     return Promise.reject(error)
   }
 }
+
+export const jwtAuthDecoder: Decoder = async (token: string, type: string) => {
```

Comment:
If anyone has a better name than this in mind, let me know. The setup command still renames it to `authDecoder`, like so, so it's not super important:

```js
// ./api/src/functions/graphql.ts
import { jwtAuthDecoder as authDecoder } from '@redwoodjs/auth-clerk-api'
```

Reply:
`clerkJwtDecoder`? Makes it clear that it specifically looks for the clerk issuer

Reply:
@dac09 went with `clerkAuthDecoder` just to keep the `authDecoder` theme